docs/db/changelog/changesets/powerauth-java-server/1.5.x/20230824-update-signature-metadata-columns.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <!-- Add new column 'signature_metadata' -->
+    <changeSet id="1" logicalFilePath="powerauth-java-server/1.5.x/20230824-update-signature-metadata-columns"
+               author="Jan Dusil">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="pa_signature_audit" columnName="signature_metadata"/>
+            </not>
+        </preConditions>
+        <comment>Add signature_metadata column of type JSON</comment>
+        <addColumn tableName="pa_signature_audit">
+            <column name="signature_metadata" type="JSONB"/>
+        </addColumn>
+    </changeSet>
+
+    <!-- Migrate data from old columns to the new JSON column -->
+    <changeSet id="2" logicalFilePath="powerauth-java-server/1.5.x/20230824-update-signature-metadata-columns"
+               author="Jan Dusil">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <columnExists tableName="pa_signature_audit" columnName="signature_data_method"/>
+                <columnExists tableName="pa_signature_audit" columnName="signature_data_uri_id"/>
+            </and>
+        </preConditions>
+        <comment>Migrate data from signature_data_method and signature_data_uri_id to signature_metadata</comment>
+        <sql>
+            UPDATE pa_signature_audit
+            SET signature_metadata = jsonb_build_object(
```

Comment:
This SQL query will only work in PostgresSQL. What about Oracle? I suggest to drop this migration.